Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Configuration.rst

```diff
@@ -62,6 +62,10 @@ Directory						Description
 *Configuration/Production/*		Configuration for the ``Production`` context
 ============================	==================================================
 
+.. note::
+	Setting Up Context with Virtual Host and change Context from «Development» to «Production» is explained in the next previous chapter «Installation».
```

Comment:
Sorry, have overlooked in this (second) PR. Sequence is «Introduction, Requirements, Installation, Configuration, Modeling, ...»
This part is in the chapter Configuration, so I think «next» is wrong. It should written:

Setting Up Context with Virtual Host and change Context from «Development» to «Production» is explained in the previous chapter «Installation».